app/experimenter/nimbus-ui/src/lib/experiment.test.ts

```diff
@@ -58,6 +58,21 @@ describe("getStatus", () => {
 
     experiment.publishStatus = NimbusExperimentPublishStatusEnum.WAITING;
     expect(getStatus(experiment).waiting).toBeTruthy();
+
+    experiment.publishStatus = NimbusExperimentPublishStatusEnum.DIRTY;
+    expect(getStatus(experiment).dirty).toBeTruthy();
+
+    experiment.publishStatus = NimbusExperimentPublishStatusEnum.REVIEW;
+    experiment.status = NimbusExperimentStatusEnum.LIVE;
+    experiment.statusNext = NimbusExperimentStatusEnum.LIVE;
+    experiment.isRollout = true;
+    expect(getStatus(experiment).updateRequested).toBeTruthy();
+
+    experiment.publishStatus = NimbusExperimentPublishStatusEnum.APPROVED;
+    expect(getStatus(experiment).updateRequestedApproved).toBeTruthy();
+
+    experiment.publishStatus = NimbusExperimentPublishStatusEnum.WAITING;
+    expect(getStatus(experiment).updateRequestedWaiting).toBeTruthy();
```

Comment:
Oh wow this is all in one test?  Woof.  Oh well it's already the pattern.

Reply:
I know, I'd like to break them out at some point :/